docker/docker-compose.yml

```diff
@@ -43,6 +43,7 @@ services:
       AzureWebJobsStorage: "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azurite:10000/devstoreaccount1;QueueEndpoint=http://azurite:10001/devstoreaccount1;TableEndpoint=http://azurite:10002/devstoreaccount1;"
       AZURE_FUNCTIONS_ENVIRONMENT: "Development"
       WEBSITE_HOSTNAME: "localhost:8080"
+      APPINSIGHTS_INSTRUMENTATIONKEY: "dummy" # necessary to get host to configure telemetry client even when just running locally
```

Comment:
Can this be an empty string?

Reply:
Let's use another string that's obviously not a valid instrumentation key, like empty GUID: `00000000-0000-0000-0000-000000000000`.

Policheck will flag this.